fedn/cli/run_cmd.py

```diff
@@ -47,8 +47,9 @@ def run_cmd(ctx):
 @click.option("-p", "--path", required=True, help="Path to package directory containing fedn.yaml")
 @click.option("-i", "--input", required=True, help="Path to input model" )
 @click.option("-o", "--output", required=True,help="Path to write the output JSON containing validation metrics")
+@click.option("-v", "--venv", default=True,type=bool,required=False, help="flag if set to False doesn't remove venv")
```

Comment:
I think it's clearer to call the boolean flag --remove-venv

Reply:
Yeah, guess this name for the flag is more suitable